config/configuration.go

```diff
@@ -274,6 +274,11 @@ func (c *Configuration) GetEthAuthKeyPair() (pub, priv string) {
 	return c.GetString("keys.ethauth.publicKey"), c.GetString("keys.ethauth.privateKey")
 }
 
+// IsPProfEnabled returns true if the pprof is enabled
+func (c *Configuration) IsPProfEnabled() bool {
```

Comment:
To follow same convention as you started earlier should it be c.GetBool(key) and then add a method that does that casting?